src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateInductionTest.kt

```diff
@@ -107,6 +108,34 @@ class CreateInductionTest : IntegrationTestBase() {
       .hasUserMessage("An Induction already exists for prisoner $prisonNumber")
   }
 
+  @Test
+  fun `should fail to create induction given database exception`() {
```

Comment:
Really we should be validating our inputs a little better than we are, but as it stands at the moment we can cause this database error at will because we are not validating this particular field (and it was this field in the request that the app sec field used which caused the problem)

I'll look at improving our request body validation in a subsequent PR, but doing so will likely mean this test no longer works as written, and it may be more difficult to create a database exception like this. Will have to give it some thought .... 🤔